index.bs

```diff
@@ -0,0 +1,505 @@
+<pre class=metadata>
+Title: Writable Files
+Shortname: WritableFiles
+Abstract: This document defines a web platform API that lets websites gain write access to the
+  native file system. It builds on [[FILE-API]], but adds lots of new functionality on top.
+Status: ED
+ED: https://wicg.github.io/writable-files/
+Level: 1
+Editor: Marijn Kruisselbrink, Google, mek@chromium.org, w3cid 72440
+Group: WICG
+Repository: wicg/writable-files
+Indent: 2
+Complain About: accidental-2119 yes, missing-example-ids yes
+Markup Shorthands: css no, markdown yes
+</pre>
+
+<pre class=link-defaults>
+spec:fetch; type:interface; text:ReadableStream
+spec:infra; type:dfn; for:/; text:set

```

Comment:
You should be able drop this if you use `[=/set=]` (at least, I was able to in one of my specs)

Reply:
Ah yes, I thought I tried that but must not have (and must have gotten confused by bikeshed showing multiple options for `/set` when I was just using `[=set=]`. But I guess the other one isn't exported so doesn't get picked up. Done.